app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class UserSerializer < ActiveModel::Serializer
+  attributes :id, :name, :display_name, :github_login, :github_url
```

Comment:
Do all of these attributes need to be included? It's easier to add to an API than it is to remove from one.